src/main/java/tutorlink/componentpackage/Exam.java

```diff
@@ -0,0 +1,11 @@
+package tutorlink.componentpackage;
+
+public class Exam extends Component{
+    public Exam(String name, double maxScore, double weight) {
+        super(name, maxScore, weight);
+    }
+    @Override
+    public String toString() {
+        return "Exam " + super.toString();
```

Comment:
Maybe for this you would want to include some formatting to make it clear that "Exam" is the description (component) and the following string is the name: 

Eg. "Exam: CS2113 finals"